README.md

```diff
@@ -3,20 +3,11 @@
 
 *Copyright © 2009–2018 The Spyder Doc Contributors*
 
-
-[![license](https://img.shields.io/pypi/l/spyder.svg)](./LICENSE.txt)
```

Comment:
The docs are licensed MIT as well, so we should leave this.

Reply:
Ok, I'll leave this one then.